web_app.py

```
@@ -3111,15 +3064,27 @@ def _settlement_period_entry(game, *, write_cm: Callable[[Any], Any]):
         _auto_close_open_night_gate_free(game, inflight_wait_s=0.0)
         with write_cm(game):
             yield
-            # with 体无异常结束（含函数 return）→ 成功，保留展示态/由推进清快照。
+            # with 体无异常结束（含函数 return）→ 成功。
```

Ruff (0.16.1)

[warning] Comment contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF003)